recipes/mmft/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -ex
+
+RUST_BACKTRACE=full
+
+if [ "$(uname)" == "Darwin" ]; then
+    # apparently the HOME variable isn't set correctly, and circle ci output indicates the following as the home directory
+    export HOME="/Users/distiller"
+    export HOME=`pwd`
```

Comment:
What's the idea of exporting HOME twice?
Only the second will survive.

Reply:
Is it really needed ?
How does it fail without this code ?

Reply:
I've removed so we will see.